spec.bs

```diff
@@ -1615,6 +1643,14 @@ they are strong, cross-site identifiers. However in order to obtain them from
 the [=Relying Party=], the merchant already needs an as-strong identifier to
 give to the [=Relying Party=] (e.g., the credit card number).
 
+## Browser Bound Key as a tracking vector ## {#sctn-privacy-browser-bound-key-tracking-vector}
+
+The BBK public key is returned on each SPC payment assertion. However, to
+obtain the BBK a merchant must obtain the credential ID from the [=Relying
+Party=] in order to initiate SPC and the user must authenticate the transaction
+using their passkey. Since accessing the BBK public key requires access to the
+credential ID first, the BBK does not add any incremental tracking.
+
```

Comment:
I think it could be useful to remind the reader here that the BBK is **only** available through SPC, and SPC necessarily involves a user interaction before the BBK is used.